app/views/users/new.html.slim

```diff
@@ -0,0 +1,23 @@
+h1 class="text-secondary mt-5" = t('auth.signup')
+
+= form_with(model: @user, local: true, url: signup_path) do |form|
+  - if @user.errors.any?
+    ul id="errorExplanation" class="alert alert-danger text-danger pl-5"
```

Comment:
Should we move this part to another partial?
https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials

And in the future, you can share it in other views

Reply:
Fixed on a8441afaf0c0a64d31d39f7b5c28089f122ff8f1